### ISSUE TITLE
Trackpad driver issue?

### ISSUE THREAD

**Issue by gjbosbert** [redorescue/redorescue#73]
I have to hold down the left hand button to be able to make the pointer move. Have checked with a USB mouse and all is fine. Could this be a trackpad driver issue?

Geo 3 13.1"

**Comment by zebradots** [comment]
@gjbosbert: Yes, this could very well be the issue. Please try the 4.0.0 release based on Debian 11 to see if this issue is resolved. It should be available within hours.
